inst/stan/wwinference.stan

```diff
@@ -223,10 +229,15 @@ transformed parameters {
   growth_site = initial_growth + eta_growth * sigma_growth; // site level growth rate
 
   // Site level spatial Rt------------------------------------------------------
-  non_norm_omega = exponential_decay_corr_func(dist_matrix, phi, l);
+  if (ind_corr_func == 1){
```

Comment:
Essentially this is your spatial/non-spatial switch correct? And currently, the way they user would specify that would be whether or not they pass in a distance matrix? I think this is fine for now, I think we might want to think about making it a bit more explicit from the user side e.g. toggling spatial component/type of correlation function on and off regardless of the presence of the distance data but I think thats for another issue.